plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/TableSnapshot.java

```diff
@@ -158,6 +159,11 @@ public List<DeltaLakeTransactionLogEntry> getJsonTransactionLogEntries()
         return logTail.getFileEntries();
     }
 
+    public Map<Long, List<DeltaLakeTransactionLogEntry>> getJsonTransactionLogVersionAndEntries()
```

Comment:
Do we rely on map ordering?

Reply:
Updated `activeAddEntries` to use `sorted(comparingByKey())`.

Reply:
i think we don't need to sort this, since it comes sorted. maybe we just keep it as a list of something
`List<Transaction>`
where "Transaction" has `long transactionId` and `List<DeltaLakeTransactionLogEntry>`?